packages/common/src/components/Input/index.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import Input from './';
+import { ThemeDecorator } from '../../stories/decorators';
+
+const stories = storiesOf('components/Input', module).addDecorator(
+  ThemeDecorator
+);
+
+stories
+  .add('Basic Input', () => <Input />)
+  .add('Block Input', () => <Input />)
```

Comment:
Are these two stories supposed to be identical?

Reply:
Nooo, fixing it